pkg/types/interfacetests/utils.go

```diff
@@ -0,0 +1,92 @@
+package interfacetests
+
+import (
+	"fmt"
+	"math/big"
+	"sort"
+	"testing"
+
+	"github.com/smartcontractkit/libocr/commontypes"
+)
+
+type BasicTester interface {
+	Setup(t *testing.T)
+	Teardown(t *testing.T)
```

Comment:
I think you can simplify this away by just letting `Setup` register things via `t.Cleanup`.  Should allow for simpler implementations too, since you can use a closure instead of having to store fields on a struct for another method to access.
```suggestion
```